.github/workflows/nix.yml

```diff
@@ -0,0 +1,37 @@
+name: "CI build"
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest-8-cores
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    steps:
+    - uses: actions/checkout@v4
+    - uses: DeterminateSystems/nix-installer-action@v10
+    - name: Authenticate to Google Cloud Platform
+      uses: google-github-actions/auth@v2
+      with:
+        project_id: yorick-dev-416917
+        workload_identity_provider: projects/752785843927/locations/global/workloadIdentityPools/github/providers/github-actions-workload-prvdr
+        service_account: github-actions-service-account@yorick-dev-416917.iam.gserviceaccount.com
+    - name: Set up cache
+      uses: zombiezen/setup-nix-cache-action@v0.3.2
+      with:
+        substituters: gs://replicate-nix-cache-dev
+        secret_keys: ${{ secrets.NIX_PRIVATE_KEY }}
+        use_nixcached: true
+    - name: Get ssh key from secret
+      env:
+        GIT_SSH_KEY: ${{ secrets.GIT_SSH_KEY }}
+      run: |
+        mkdir -p ~/.ssh
+        cat > ~/.ssh/id_ed25519 <<< "$GIT_SSH_KEY"
+        chmod 0600 ~/.ssh/id_ed25519
+    - run: nix build --accept-flake-config .#cog-triton-builder -o cog-triton-builder
+    - run: nix build --accept-flake-config .#cog-triton-runner-86 -o cog-triton-runner-86
+    - run: nix build --accept-flake-config .#cog-triton-runner-80 -o cog-triton-runner-80
+    - run: nix build --accept-flake-config .#cog-triton-runner-90 -o cog-triton-runner-90
```

Comment:
where is this pushing to?

Reply:
Just a build cache in gs://replicate-nix-cache-dev. Which replicate images should it be pushing to?